src/postProcessor.ts

```diff
@@ -81,6 +81,7 @@ export function buildPostProcessor(
 
         if (match) {
           li.addClass('lc-list-callout');
+          li.setAttribute('data-type', "lc-" + callout.char);
```

Comment:
```suggestion
          li.setAttribute('data-callout', callout.char);
```

Reply:
Makes sense to me! Thanks for the quick review!